reference/features/security.md

```diff
@@ -0,0 +1,272 @@
+  "actions": [
+    "search"
+  ],
+  "indexes": [
+    "*"
+  ],
+  "expiresAt": null,
+  "createdAt": "2021-08-11T10:00:00Z",
+  "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+`Default Admin API Key`, instead, has access to all API endpoints except `/keys`:
+
+```json
+{
+  "description": "Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)",
+  "key": "380689dd379232519a54d15935750cc7625620a2ea2fc06907cb40ba5b421b6f",
+  "actions": [
+    "*"
+  ],
+  "indexes": [
+    "*"
+  ],
+  "expiresAt": null,
+  "createdAt": "2021-08-11T10:00:00Z",
+  "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+Both default API keys have access to all indexes in an instance and do not have an expiry date. If necessary, they can be deleted or edited like any user-created key.
+
+### Listing API keys
+
+You can use the [`/keys` endpoint](/reference/api/keys.md) to obtain information on any active key in your MeiliSearch instance. This can be useful when you need an overview of which keys exist and what permissions they have.
+
+[`GET /keys`](/reference/api/keys.md#get-all-keys) returns a full list of all existing keys. Expired and deleted keys will not appear in the response. As with creating, deleting, and updating API keys, you need the `master` key to access this endpoint.
+
+[`GET /keys/:key`](/reference/api/keys.md#get-one-key), where `:key` stands for the full value of an active key, returns information on a single key.
+
+#### Example
+
+You can query your instance to confirm which active keys have access to your `scifi_books` index:
+
+<CodeSamples id="security_guide_list_keys_1" />
+
+```json
+{
+  "results": [
+    {
+      "description": "Default Search API Key (Use it to search from the frontend code)",
+      "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99",
+      "actions": [
+          "search"
+      ],
+      "indexes": [
+          "*"
+      ],
+      "expiresAt": null,
+      "createdAt": "2021-08-11T10:00:00Z",
+      "updatedAt": "2021-08-11T10:00:00Z"
+    },
+    {
+      "description": "Default Admin API Key (Use it for all other operations. Caution! Do not share it on the client side)",
+      "key": "380689dd379232519a54d15935750cc7625620a2ea2fc06907cb40ba5b421b6f",
+      "actions": [
+          "*"
+      ],
+      "indexes": [
+          "*"
+      ],
+      "expiresAt": null,
+      "createdAt": "2021-08-11T10:00:00Z",
+      "updatedAt": "2021-08-11T10:00:00Z"
+    }
+  ]
+}
+```
+
+### Updating an API key
+
+You can freely update an API key until it expires. This includes editing the indexes, endpoints, and routes it can access, as well as its description and expiry date.
+
+To update an API key, you must use the [`PATCH /keys/:key`](/reference/api/keys.md#update-a-key) endpoint which can only be accessed with the `master` key. Here, `:key` stands for the full value of the key you want to update.
```

Comment:
I still think we should follow a single convention for linking to API reference pages, and that we should prefer our name for the endpoint over the HTTP verb + route. (Although I realize that it sounds kind of funny and redundant here.)

```suggestion
To update an API key, you must use the [update API key endpoint](/reference/api/keys.md#update-a-key) which can only be accessed with the `master` key. `:key` stands for the full value of the key you want to update.
```